<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cImportIngredientDialog</class>
 <widget class="QDialog" name="cImportIngredientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Ingredient</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0" colspan="2">
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeView" name="m_lpIngredientList">
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
       </widget>
       <widget class="QTreeView" name="m_lpIngredientDetails"/>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Source:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_lpSource">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <widget class="QDialogButtonBox" name="m_lpButtonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="m_lpSearchString"/>
       </item>
       <item>
        <widget class="QPushButton" name="m_lpSearchButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Ingredient Group:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_lpIngredientGroup"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_lpButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>cImportIngredientDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_lpButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>cImportIngredientDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
